### PR TITLE
Certificate of Status API and UI changes

### PR DESCRIPTION
**Pull request by lekshmimallika-aot**
*Issue #:* /bcgov/entity#12677, /bcgov/entity#12679,  /bcgov/entity#13101

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: search-ui/src/views/BusinessInfoView.vue [hunk 1]
```diff
@@ -258,10 +258,25 @@ const loadPurchasableDocs = async () => {
...
+  }
 }
 
 const isCogsAvailable = () => {
-  return !isFirm.value
+  return !isFirm.value && isActive.value
+}
+
+const isCstatAvailable = () => {
+  return (isBC.value || isCoop.value) && isActive.value
```
Comment (kialj876) [inline search-ui/src/views/BusinessInfoView.vue:279]: should this include BENs too?

Comment (lekshmimallika-aot) [reply search-ui/src/views/BusinessInfoView.vue:279]: BEN is not in the list provided

Comment (kialj876) [reply search-ui/src/views/BusinessInfoView.vue:279]: Oh weird. I thought BENs were essentially the exact same as BCs except for the benefit statement

**Review by kialj876 — APPROVED** [review]
Looks good!

File: search-ui/src/views/BusinessInfoView.vue (hunk 1 above)
Comment (lekshmimallika-aot) [reply search-ui/src/views/BusinessInfoView.vue:279]: I will confirm it with Jenny

**Pull request merged**